### PR TITLE
Announce Overview and Onboarding screens - accessibility (EXPOSUREAPP-4454-4455)

### PR DESCRIPTION
**Pull request by chiljamgossow**
Announce Overview and Onboarding screens
Use toolbar

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -66,6 +66,16 @@
         <item name="android:tint">@color/colorTextPrimary1</item>
     </style>
 
+    <style name="CWAToolbar.BackArrow">
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/values/styles.xml:69]: Should this have  `CWAToolbar` as parent?

File: Corona-Warn-App/src/main/res/values/contact_diary_strings.xml [hunk 2]
```diff
@@ -60,7 +60,7 @@
     <!-- XTXT: Subtitle for contact diary overview screen -->
     <string name="contact_diary_overview_subtitle">"Write down the people you have met and places you have visited."</string>
     <!-- XTXT: Header for contact diary overview screen -->
-    <string name="contact_diary_overview_header">"Start Page"</string>
+    <string name="contact_diary_overview_header">Start Page</string>
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/values/contact_diary_strings.xml:63]: Why this change? Do we need to change this in other locales too?

File: Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml [hunk 3]
```diff
@@ -4,35 +4,35 @@
...
+        android:id="@+id/content_container"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
+        android:layout_height="match_parent"
+        android:contentDescription="@string/contact_diary_overview_title">
 
-        <include
-            android:id="@+id/contact_diary_overview_header"
-            layout="@layout/include_contact_diary_overview_header"
+        <androidx.appcompat.widget.Toolbar
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml:12]: Toolbar in Overview Screen should be completely black instead of grey when using dark mode

File: Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml [hunk 4]
```diff
@@ -5,20 +5,20 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/content_container"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@drawable/contact_diary_onboarding_background">
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml:10]: Why is this removed? Background should be greyish in dark mode in the onboarding screen (not only toolbar)

File: Corona-Warn-App/src/main/res/values/styles.xml (hunk 1 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/values/styles.xml:69]: yes

File: Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml (hunk 3 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml:12]: We have a design for that? The app is now consistent in dark mode, before it was not

Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/layout/contact_diary_overview_fragment.xml:12]: Ok, I will change it to black for this screen.

File: Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml (hunk 4 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/layout/contact_diary_onboarding_fragment.xml:10]: I put it back although the different style does not make sense to

File: Corona-Warn-App/src/main/res/values/contact_diary_strings.xml (hunk 2 above)
Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/values/contact_diary_strings.xml:63]: That just struck my eye in the preview, seems fine on device

Comment (d4rken) [reply Corona-Warn-App/src/main/res/values/contact_diary_strings.xml:63]: Note that any changes made to these strings will likely be overwritten by the next time translations are pulled.

**Review by d4rken — APPROVED** [review]

**Review by axelherbstreith — APPROVED** [review]
LGTM
